.github/workflows/gaia.yml

```diff
@@ -23,6 +23,12 @@ jobs:
   gaia-test:
     runs-on: [self-hosted, manylinux2014]
     steps:
+    - name: Detect the tmate session
+      if: "jobs.gaia-test.steps[*].uses == 'mxschmitt/action-tmate@*'"
```

Comment:
could we use '*/action-tmate'?
Since mxschmitt is not the only one that provides action-tmate.

Reply:
Make sense to me.